ENGAGEHF/Dashboard/Notifications/Views/NotificationRow.swift

```diff
@@ -0,0 +1,111 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct NotificationRow: View {
+    @Binding var notification: Notification
```

Comment:
I think that we can remove the `@Binding` here and in some similar views. There won't be any backward communication from here to the upper views and it seems like the notification can be a constant property as it will be never changed and modified which would need to be communicated up to parent views.